content-src/components/TopSites/TopSites.js

```diff
@@ -12,14 +13,18 @@ const TopSites = React.createClass({
     return (<section className="top-sites">
       <h3 className="section-title">Top Sites</h3>
       <div className="tiles-wrapper">
-        {sites.map((site) => (<a key={site.url} className="tile" href={site.url}>
-          <div className="tile-img-container">
-            <SiteIcon site={site} width={100} height={100} />
-          </div>
-          <div className="tile-title">
-            {site.provider_name}
-          </div>
-        </a>))}
+        {sites.map((site) => {
+          const color = site.favicon_colors && site.favicon_colors[0] && site.favicon_colors[0].color || [333, 333, 333];
```

Comment:
Wait, isn't 255 == FF? Not sure what color `[333, 333, 333]` would be (whiter than white), or if these should be `[33, 33, 33]` (darker gray).